docs/functionality/tenant_mgmt.md

```diff
@@ -0,0 +1,69 @@
+﻿# Tenant/Facility Management
+
+In Link Cloud, the terms "tenant" and "facility" are used interchangeably to refer to healthcare organizations configured within the system. Each tenant/facility is identified by a unique `facilityId` that must be consistent across all services to ensure proper functionality.
+
+## Tenant Service
+The Tenant service acts as the primary entry point for configuring organizations within Link Cloud. It maintains core facility configurations and generates events for scheduled measure reporting periods.
+
+### Core Tenant Configuration Elements
+- Facility ID (unique identifier used across all services)
+- Facility Name
+- Creation and Modification dates
+- Scheduled Tasks
+- Monthly Reporting Plans
+    - Report Types
+    - Reporting Periods
+    - Submission Schedules
+
+## Cross-Service Tenant Configuration
+
+A fully functional tenant requires configuration across multiple services. Each service maintains its own tenant-specific configurations using the `facilityId` as the linking identifier.
+
+### Service-Specific Configurations
```

Comment:
Query Dispatch is missing from this list. It should probably be added until it is officially phased out.

Reply:
Census service also looks like it's missing